src/content/end.js

```diff
@@ -287,16 +287,26 @@ class Main extends Logged {
 
     constructor() {
         super()
+        this.orderC = 1000
+    }
 
+    async initialise() {
         const subjects = document.createElement('div')
         subjects.classList.add('subjectSelect')
 
         const settings = document.createElement('div')
         settings.classList.add('subjectSelect')
         settings.classList.add('mainInfo');
 
-        this.orderC = 1000
         let orders = {}
+        
+        // get subject URLs from courses
+        const subjectInfo = await fetch("https://courses.fit.cvut.cz/data/courses-all.json", {
+            method: 'GET',
+            mode: 'cors',
+            credentials: 'omit'
+        }).then(response => response.json())
+            .catch(error => console.error(error))
```

Comment:
This seems to be the only console log it the entire app. If any errors occur, we should probably try to deal with them (or in this case we may just ignore them).

Reply:
Fixed this in the newest commits.